src/python/grpcio/tests/unit/_adapter/_intermediary_low_test.py

```diff
@@ -67,7 +69,7 @@ def testLonelyClient(self):
     second_event = completion_queue.get(after_deadline)
     self.assertIsNotNone(second_event)
     kinds = [event.kind for event in (first_event, second_event)]
-    self.assertItemsEqual(
+    six.assertCountEqual(self,
```

Comment:
I find it *so* weird that when Python 2 became Python 3 *this* is [how things ended up renamed](https://hg.python.org/cpython/rev/596239da3db7/#l2.15). Just... so weird (`Items`? `Count`? Ehhh??). Thanks for this, though.